### ISSUE TITLE
saveAs Option - String

### ISSUE THREAD

**Issue by mattmawhinney** [balderdashy/skipper#44]
Using the saveAs option with string input returns the following: 

![screen shot 2014-10-07 at 12 09 12 pm](https://cloud.githubusercontent.com/assets/3609302/4545624/57f6a21c-4e3c-11e4-8a36-8a6ab61e7444.png)

However, if I use the filename option, my code works. 

![screen shot 2014-10-07 at 12 11 05 pm](https://cloud.githubusercontent.com/assets/3609302/4545652/a6d52994-4e3c-11e4-9803-dfabe356b952.png)

I'd suggest adding filename in the doc under options.

**Comment by sailsbot** [comment]
Thanks for posting, @mattmawhinney.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
